tools/db_crashtest.py

```diff
@@ -202,6 +202,7 @@
     "max_auto_readahead_size": lambda: random.choice([0, 16384, 524288]),
     "num_file_reads_for_auto_readahead": lambda: random.choice([0, 1, 2]),
     "min_write_buffer_number_to_merge": lambda: random.choice([1, 2]),
+    "preserve_internal_time_seconds": lambda: random.choice([0, 60, 3600, 36000]),
```

Comment:
3600 is a little bit too long for stress test to trigger. Perhaps something like 10 minutes is more likely to trigger.